### ISSUE TITLE
why I use your pretrained module to test on validation set, it can not reach your result, and I use your config to train module again, it also can not reach your result?

### ISSUE THREAD

**Issue by the-king-of-yhq** [Gorilla-Lab-SCUT/VISTA#20]
use your pretrained model: mAP 0.6097 mATE 0.3028 mASE 0.2483 mAOE 0.2644 mAVE 0.2113 NDS 0.6826
use your config train again mAP  0.6077 mATE 0.2933 mASE 0.2474 mAOE 0.2600 mAVE 1.4374 NDS 0.5753
why the train model  mAVE and NDS is too bad
